fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,51 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+DoSliceTimeCorrection=`opts_GetOpt1 "--doslicetime" $@`                 # Whether to do slicetime correction (TRUE), FALSE to omit
+SliceTimerCorrectionParameters=`opts_GetOpt1 "--slicetimerparams" $@`   # A '@' separated list of FSL slicetimer options. Please see FSL slicetimer documentation for details.
+                                                                        # Verbose (-v) is already turned on. TR (-r) is specified using --tr parameter.
+TR=`opts_GetOpt1 "--tr" $@`                                             # TR of the timeseries
```

Comment:
I believe in all our other scripts in which we need to know the TR, we extract it from the NIFTI directly using something like:
```
TR=`${FSLDIR}/bin/fslval ${fMRIName} pixdim4`
```
So, I'd suggest we do the same within the `$DoSliceTimeCorrection` conditional.

Reply:
Ok.